raster/r.sim/r.sim.sediment/r.sim.sediment.md

```diff
@@ -26,10 +26,10 @@ reference literature (see suggested values in Notes and References).
 
 Output includes transport capacity raster map *transport_capacity* in
 \[kg/ms\], transport capacity limited erosion/deposition raster map
-*tlimit_erosion_deposition* \[kg/m<sup>2</sup>s\]i that are output
+*tlimit_erosion_deposition* \[kg/m^2 s\]i that are output
 almost immediately and can be viewed while the simulation continues.
 Sediment flow rate raster map *sediment_flux* \[kg/ms\], and net
-erosion/deposition raster map \[kg/m<sup>2</sup>s\] can take longer time
+erosion/deposition raster map \[kg/m^2s\] can take longer time
```

Comment:
`kg/m^2 s` or `kg/m^2s`? Sometimes there's a dot or hyphen in these kinds of units to know which is grouped in the division, or brackets, but not that often.
(I _don't_ think it is `kg/(m^(2*s))` from the context around though)


There's also the unit in the middle that is ambiguous: `kg/ms`: kilogram per millisecond, kilogram per meter-second?

Reply:
I tried to keep the style there, the space in there was unintended. So at least it's consistent. I think it's clear it's not millisecond from the context.

Reply:
And there was an extra "i", I removed it as typo, it was in the original html as well.